.github/workflows/daily-test.yaml

```diff
@@ -0,0 +1,49 @@
+name: Daily build
+
+on:
+  schedule:
+    # Run daily at 00:00
+    - cron: '0 0 * * *'
+  push:
+  pull_request:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+        python-version: [3.8]
+        c_compiler: [gcc, gcc-5]
+        cpp_compiler: [g++, g++-5]
```

Comment:
Default gcc/g++ will be tested in the regular build run. Adding it here makes it a bit redundant. Other OSes and their non-standard compilators would be interesting though, but maybe that's out of scope for this PR.

Reply:
True, I'll remove `gcc` from the building matrix. What other combinations would you suggest?